packages/console/src/mdx-components/Step/index.tsx

```diff
@@ -21,7 +22,7 @@ function Step({ title, subtitle, index, children }: Props, ref?: Ref<HTMLDivElem
         <CardTitle
           size="medium"
           title={<DangerousRaw>{title}</DangerousRaw>}
-          subtitle={<DangerousRaw>{subtitle}</DangerousRaw>}
+          subtitle={cond(subtitle && <DangerousRaw>{subtitle}</DangerousRaw>)}
```

Comment:
should we update the `DangerousRaw` component instead?

Reply:
Yes, it's better that way. Updated